catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryMetacardHandler.java

```diff
@@ -92,13 +109,18 @@ private void processMetacardCreateUpdate(Metacard mcard) throws FederationAdminE
         if (CollectionUtils.isEmpty(metacards)) {
             prepareMetacard(mcard, null);
             federationAdminService.addRegistryEntry(mcard);
-            LOGGER.debug("Successfully created new registry metacard");
         } else if (shouldUpdate(mcard, metacards.get(0))) {
             prepareMetacard(mcard,
                     metacards.get(0)
                             .getId());
             federationAdminService.updateRegistryEntry(mcard);
-            LOGGER.debug("Successfully update registry metacard");
+        } else {
+            LOGGER.debug(
+                    "New remote registry metacard {} is older ({}) than current local version ({})",
```

Comment:
❓  Should we include that the update action will not performed if this is the case?